docs/shrinkwrap.md

```diff
@@ -32,12 +32,13 @@ important that your node_modules is deleted before you do this to be sure to pic
 
 - Install [shonkwrap](https://github.com/skybet/shonkwrap) globally: `npm install -g shonkwrap`. This package attempts
 to remove the from/resolved fields if possible.
+- (Optional) Modify package.json. For example: `npm install --save lodash@4.11.1` or `npm uninstall --save left-pad`
```

Comment:
If installing a specific version while shrinkwrap is still present, will it actually install that version? Or the version from shrinkwrap?